### PR TITLE
[RW-8892][risk=low] Add admin endpoint to get and create Egress bypass window

### PR DESCRIPTION
**Pull request by yonghaoy**
How UI will use this endpoint: Admin can add user bypass window in admin/user page to allow user download large file. 
Doc: https://docs.google.com/document/d/1hM1ODnAEzWQMrueXmDsYHePft5s9eLAsmq_Fh78D4Cg/edit?resourcekey=0-pZQ4jtyKEVnr7pezJ0oa7w

---
**PR checklist**

- [ ] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have manually tested this change and my testing process is described above.
- [ ] This change includes appropriate automated tests, and I have documented any behavior that cannot be tested with code.
- [ ] I have added explanatory comments where the logic is not obvious.
- One or more of the following is true:
  - [ ] This change is intended to complete a JIRA story, so I have checked that all AC are met for that story.
  - [ ] This change fixes a bug, so I have ensured the steps to reproduce are in the Jira ticket or provided above.
  - [ ] This change impacts deployment safety (e.g. removing/altering APIs which are in use), so I have documented the impacts in the description.
  - [ ] This change includes a new feature flag, so I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later.
  - [ ] This change modifies the UI, so I have taken screenshots or recordings of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P).
  - [ ] This change modifies API behavior, so I have run the relevant E2E tests locally because API changes are not covered by our PR checks.
  - [ ] None of the above apply to this change.

### CODE REVIEW

File: api/src/main/java/org/pmiops/workbench/user/UserAdminService.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      UserEgressBypassWindowDao userEgressBypassWindowDao,
+      EgressBypassWindowMapper egressBypassWindowMapper,
+      Clock clock) {
+    this.userEgressBypassWindowDao = userEgressBypassWindowDao;
+    this.egressBypassWindowMapper = egressBypassWindowMapper;
+    this.clock = clock;
+  }
+
+  public void createEgressBypassWindow(Long userId, Instant startTime, String description) {
+    Instant endTime = startTime.plus(2, ChronoUnit.DAYS);
```
Comment (tarekmamdouh) [inline api/src/main/java/org/pmiops/workbench/user/UserAdminService.java:33]: I assume 2 days is a requirement?

File: api/src/main/java/org/pmiops/workbench/user/UserAdminService.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  public EgressBypassWindow getCurrentEgressBypassWindow(Long userId) {
+    Set<DbUserEgressBypassWindow> dbUserEgressBypassWindows =
+        userEgressBypassWindowDao.getByUserIdOrderByStartTimeDesc(userId);
+    getActiveEgressBypassWindow(dbUserEgressBypassWindows, clock.instant());
+    return egressBypassWindowMapper.toApiEgressBypassWindow(
+        getActiveEgressBypassWindow(dbUserEgressBypassWindows, clock.instant()).orElse(null));
+  }
+
+  /**
+   * Returns {@code true} if current timestamp is between any {@code DbUserEgressBypassWindow} start
```
Comment (tarekmamdouh) [inline api/src/main/java/org/pmiops/workbench/user/UserAdminService.java:51]: nitpik: returns the bypass window as an Optional.

**Review by tarekmamdouh — APPROVED** [review]
LGTM

File: api/src/main/java/org/pmiops/workbench/user/UserAdminService.java (hunk 1 above)
Comment (yonghaoy) [reply api/src/main/java/org/pmiops/workbench/user/UserAdminService.java:33]: Yup, it is from Moira's doc: https://docs.google.com/document/d/1hM1ODnAEzWQMrueXmDsYHePft5s9eLAsmq_Fh78D4Cg/edit?resourcekey=0-pZQ4jtyKEVnr7pezJ0oa7w
I will make this a constant.

**Pull request merged**